engine/class_modules/monk/sc_monk.cpp

```diff
@@ -327,16 +344,159 @@ struct monk_action_t : public Base
       p()->cooldown.bonedust_brew->adjust( timespan_t::from_seconds( time_reduction ), true );
   }
 
+   int get_bdb_zone()
```

Comment:
First, this function should be const, and the ones deriving from it as well.

Second: Is it really best to return an int instead of a enum that naturally restricts the possible zones that can be returned?